src/soorgeon/cli.py

```diff
@@ -96,3 +99,54 @@ def clean(filename):
 
     """
     basic_clean(filename)
+
+
+@cli.command()
+@click.argument("filename", type=click.Path(exists=True))
+def test(filename):
+    """
+    check if a .py or .ipynb file runs.
+
+    $ soorgeon run path/to/script.py
```

Comment:
`soorgeon test`?

Reply:
Will fix it. Thanks for pointing that out.